alpenhorn/auto_import.py

```diff
@@ -470,6 +533,22 @@ def _import_file(node, root, acq_name, file_name):
                     "empty. Leaving fields NULL." % (acq_name)
                 )
                 di.CorrAcqInfo.create(acq=acq)
+    if atype == "hfb" and ftype.name == "hfb":
+        if not acq.corrinfos.count():
+            try:
+                di.CorrAcqInfo.create(
```

Comment:
```suggestion
                di.HFBAcqInfo.create(
```
(and throughout).  The `CorrAcqInfo` table is for files of type `corr`.